### DISCUSSION TITLE
How to open external links from content in a new window

### DISCUSSION THREAD

**Discussion by andreaspreuss** [typemill/typemill#489] [Q&A]
Hello and best regards,
is there an easy possibility to open external links from content in a new window?
I tried it with Github Markdown `[name](link){:target=“_blank”}` but unfortunately it does not work.
Is there another way? - I also tried - `[Document](http://www.otherdocusample.com){:target="_blank" rel="noopener"}`
Or should there be an Javascript Overlay in the Template?
` $(this).attr({
               target: "_blank",
               title: "Opens in a new window"
           });`
I don't want to spend too much time fiddling around and perhaps invent the wheel a second time. I also couldn't find any information about this in the documentation. I bought a license yesterday and would be very happy about a solution about it.
Thank you very much. 

Kind regards

**Comment by trendschau** [comment]
Hi Andreas, 

thank you for supporting Typemill! There was a discussion about this already with solutions in this thread: https://github.com/typemill/typemill/issues/405 So 

`[Document](http://www.otherdocusample.com){target="_blank"}`

should work (without the colon). I will add it to the documentation.

Best regards
